mini-kodein-android-compose/proguard-rules.pro

```diff
@@ -18,4 +18,4 @@
 
```

Comment:
Don't we need any kind of specific rule for Compose?

Reply:
We only use the navigation library for compose, and it has its own proguard file, so I think we don't need any